va/va_drmcommon.h

```diff
@@ -155,4 +160,23 @@ typedef struct _VADRMPRIMESurfaceDescriptor {
 } VADRMPRIMESurfaceDescriptor;
 
 
+typedef struct _VADRMPRIMESurfaceDescriptorExt {
+    VADRMPRIMESurfaceDescriptor surface_desc;
+    /* memory location , local memory or system memory */
+    union{
+        struct{
+            /* whether the memory is allocated on local memory */
+            uint32_t local_memory :1;
+            /* whether the memory region is enabled */
+            uint32_t memory_region_enable: 1;
+            /* memory region index */
+            uint32_t memory_region_index: 8;
```

Comment:
Index of what?  Is this actually the ID of the memory region, as in `enum intel_region_id`?

Reply:
the ID of the memory region, sorry , seems it is confusing, because there are already intel_region_id defined. so. maybe I need use other name to distinguish with enum intel_region_id

Reply:
If there is an index, then there should be a range for this index which does not seem present in this PR. This is confusing and needs a correction: please, specify how to get a range?